2018-08-01 01:49:13,255  ERROR [main] com.infy.dao.UserDAOImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:176)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:135)
	at com.infy.business.service.UserServiceImpl.deleteUser(UserServiceImpl.java:165)
	at com.infy.ui.UserInterface.deleteUser(UserInterface.java:150)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-01 01:49:13,277  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:176)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:135)
	at com.infy.business.service.UserServiceImpl.deleteUser(UserServiceImpl.java:165)
	at com.infy.ui.UserInterface.deleteUser(UserInterface.java:150)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-01 01:49:13,289  ERROR [main] com.infy.business.service.UserServiceImpl [deleteUser] - Service.USER_NOT_FOUND
java.lang.Exception: Service.USER_NOT_FOUND
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:142)
	at com.infy.business.service.UserServiceImpl.deleteUser(UserServiceImpl.java:165)
	at com.infy.ui.UserInterface.deleteUser(UserInterface.java:150)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-01 01:49:13,298  DEBUG [main] com.infy.dao.ProductDAOImpl [getProductDetails] - Unknown entity: com.infy.entity.ProductEntity
org.hibernate.MappingException: Unknown entity: com.infy.entity.ProductEntity
	at org.hibernate.internal.SessionFactoryImpl.getEntityPersister(SessionFactoryImpl.java:1096)
	at org.hibernate.internal.SessionImpl.locateEntityPersister(SessionImpl.java:2601)
	at org.hibernate.internal.SessionImpl.access$1900(SessionImpl.java:176)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2540)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2544)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2531)
	at org.hibernate.internal.SessionImpl.byId(SessionImpl.java:1079)
	at org.hibernate.internal.SessionImpl.get(SessionImpl.java:991)
	at com.infy.dao.ProductDAOImpl.getProductDetails(ProductDAOImpl.java:218)
	at com.infy.business.service.ProductServiceImpl.getProductDetails(ProductServiceImpl.java:139)
	at com.infy.ui.UserInterface.getProductDetails(UserInterface.java:227)
	at com.infy.ui.UserInterface.main(UserInterface.java:43)
2018-08-01 01:49:27,743  DEBUG [main] com.infy.dao.ProductDAOImpl [getProductDetails] - Unknown entity: com.infy.entity.ProductEntity
org.hibernate.MappingException: Unknown entity: com.infy.entity.ProductEntity
	at org.hibernate.internal.SessionFactoryImpl.getEntityPersister(SessionFactoryImpl.java:1096)
	at org.hibernate.internal.SessionImpl.locateEntityPersister(SessionImpl.java:2601)
	at org.hibernate.internal.SessionImpl.access$1900(SessionImpl.java:176)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2540)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2544)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2531)
	at org.hibernate.internal.SessionImpl.byId(SessionImpl.java:1079)
	at org.hibernate.internal.SessionImpl.get(SessionImpl.java:991)
	at com.infy.dao.ProductDAOImpl.getProductDetails(ProductDAOImpl.java:218)
	at com.infy.business.service.ProductServiceImpl.getProductDetails(ProductServiceImpl.java:139)
	at com.infy.ui.UserInterface.getProductDetails(UserInterface.java:227)
	at com.infy.ui.UserInterface.main(UserInterface.java:43)
2018-08-01 01:49:59,591  DEBUG [main] com.infy.dao.ProductDAOImpl [getProductDetails] - Unknown entity: com.infy.entity.ProductEntity
org.hibernate.MappingException: Unknown entity: com.infy.entity.ProductEntity
	at org.hibernate.internal.SessionFactoryImpl.getEntityPersister(SessionFactoryImpl.java:1096)
	at org.hibernate.internal.SessionImpl.locateEntityPersister(SessionImpl.java:2601)
	at org.hibernate.internal.SessionImpl.access$1900(SessionImpl.java:176)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2540)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2544)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2531)
	at org.hibernate.internal.SessionImpl.byId(SessionImpl.java:1079)
	at org.hibernate.internal.SessionImpl.get(SessionImpl.java:991)
	at com.infy.dao.ProductDAOImpl.getProductDetails(ProductDAOImpl.java:218)
	at com.infy.business.service.ProductServiceImpl.getProductDetails(ProductServiceImpl.java:139)
	at com.infy.ui.UserInterface.getProductDetails(UserInterface.java:227)
	at com.infy.ui.UserInterface.main(UserInterface.java:43)
2018-08-01 01:51:04,154  DEBUG [main] com.infy.dao.ProductDAOImpl [getProductDetails] - Unknown entity: com.infy.entity.ProductEntity
org.hibernate.MappingException: Unknown entity: com.infy.entity.ProductEntity
	at org.hibernate.internal.SessionFactoryImpl.getEntityPersister(SessionFactoryImpl.java:1096)
	at org.hibernate.internal.SessionImpl.locateEntityPersister(SessionImpl.java:2601)
	at org.hibernate.internal.SessionImpl.access$1900(SessionImpl.java:176)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2540)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2544)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2531)
	at org.hibernate.internal.SessionImpl.byId(SessionImpl.java:1079)
	at org.hibernate.internal.SessionImpl.get(SessionImpl.java:991)
	at com.infy.dao.ProductDAOImpl.getProductDetails(ProductDAOImpl.java:218)
	at com.infy.business.service.ProductServiceImpl.getProductDetails(ProductServiceImpl.java:139)
	at com.infy.ui.UserInterface.getProductDetails(UserInterface.java:227)
	at com.infy.ui.UserInterface.main(UserInterface.java:43)
2018-08-01 01:51:27,581  DEBUG [main] com.infy.dao.ProductDAOImpl [getProductDetails] - Unknown entity: com.infy.entity.ProductEntity
org.hibernate.MappingException: Unknown entity: com.infy.entity.ProductEntity
	at org.hibernate.internal.SessionFactoryImpl.getEntityPersister(SessionFactoryImpl.java:1096)
	at org.hibernate.internal.SessionImpl.locateEntityPersister(SessionImpl.java:2601)
	at org.hibernate.internal.SessionImpl.access$1900(SessionImpl.java:176)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2540)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2544)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2531)
	at org.hibernate.internal.SessionImpl.byId(SessionImpl.java:1079)
	at org.hibernate.internal.SessionImpl.get(SessionImpl.java:991)
	at com.infy.dao.ProductDAOImpl.getProductDetails(ProductDAOImpl.java:218)
	at com.infy.business.service.ProductServiceImpl.getProductDetails(ProductServiceImpl.java:139)
	at com.infy.ui.UserInterface.getProductDetails(UserInterface.java:227)
	at com.infy.ui.UserInterface.main(UserInterface.java:43)
2018-08-01 01:51:33,818  DEBUG [main] com.infy.dao.ProductDAOImpl [getProductDetails] - Unknown entity: com.infy.entity.ProductEntity
org.hibernate.MappingException: Unknown entity: com.infy.entity.ProductEntity
	at org.hibernate.internal.SessionFactoryImpl.getEntityPersister(SessionFactoryImpl.java:1096)
	at org.hibernate.internal.SessionImpl.locateEntityPersister(SessionImpl.java:2601)
	at org.hibernate.internal.SessionImpl.access$1900(SessionImpl.java:176)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2540)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2544)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2531)
	at org.hibernate.internal.SessionImpl.byId(SessionImpl.java:1079)
	at org.hibernate.internal.SessionImpl.get(SessionImpl.java:991)
	at com.infy.dao.ProductDAOImpl.getProductDetails(ProductDAOImpl.java:218)
	at com.infy.business.service.ProductServiceImpl.getProductDetails(ProductServiceImpl.java:139)
	at com.infy.ui.UserInterface.getProductDetails(UserInterface.java:227)
	at com.infy.ui.UserInterface.main(UserInterface.java:43)
2018-08-01 01:54:55,821  DEBUG [main] com.infy.dao.ProductDAOImpl [getProductDetails] - Unknown entity: com.infy.entity.ProductEntity
org.hibernate.MappingException: Unknown entity: com.infy.entity.ProductEntity
	at org.hibernate.internal.SessionFactoryImpl.getEntityPersister(SessionFactoryImpl.java:1096)
	at org.hibernate.internal.SessionImpl.locateEntityPersister(SessionImpl.java:2601)
	at org.hibernate.internal.SessionImpl.access$1900(SessionImpl.java:176)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2540)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2544)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2531)
	at org.hibernate.internal.SessionImpl.byId(SessionImpl.java:1079)
	at org.hibernate.internal.SessionImpl.get(SessionImpl.java:991)
	at com.infy.dao.ProductDAOImpl.getProductDetails(ProductDAOImpl.java:218)
	at com.infy.business.service.ProductServiceImpl.getProductDetails(ProductServiceImpl.java:139)
	at com.infy.ui.UserInterface.getProductDetails(UserInterface.java:227)
	at com.infy.ui.UserInterface.main(UserInterface.java:43)
2018-08-01 01:56:11,610  DEBUG [main] com.infy.dao.ProductDAOImpl [getProductDetails] - Unknown entity: com.infy.entity.ProductEntity
org.hibernate.MappingException: Unknown entity: com.infy.entity.ProductEntity
	at org.hibernate.internal.SessionFactoryImpl.getEntityPersister(SessionFactoryImpl.java:1096)
	at org.hibernate.internal.SessionImpl.locateEntityPersister(SessionImpl.java:2601)
	at org.hibernate.internal.SessionImpl.access$1900(SessionImpl.java:176)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2540)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2544)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2531)
	at org.hibernate.internal.SessionImpl.byId(SessionImpl.java:1079)
	at org.hibernate.internal.SessionImpl.get(SessionImpl.java:991)
	at com.infy.dao.ProductDAOImpl.getProductDetails(ProductDAOImpl.java:218)
	at com.infy.business.service.ProductServiceImpl.getProductDetails(ProductServiceImpl.java:139)
	at com.infy.ui.UserInterface.getProductDetails(UserInterface.java:227)
	at com.infy.ui.UserInterface.main(UserInterface.java:43)
2018-08-01 01:56:57,311  DEBUG [main] com.infy.dao.ProductDAOImpl [getProductDetails] - Unknown entity: com.infy.entity.ProductEntity
org.hibernate.MappingException: Unknown entity: com.infy.entity.ProductEntity
	at org.hibernate.internal.SessionFactoryImpl.getEntityPersister(SessionFactoryImpl.java:1096)
	at org.hibernate.internal.SessionImpl.locateEntityPersister(SessionImpl.java:2601)
	at org.hibernate.internal.SessionImpl.access$1900(SessionImpl.java:176)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2540)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2544)
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.<init>(SessionImpl.java:2531)
	at org.hibernate.internal.SessionImpl.byId(SessionImpl.java:1079)
	at org.hibernate.internal.SessionImpl.get(SessionImpl.java:991)
	at com.infy.dao.ProductDAOImpl.getProductDetails(ProductDAOImpl.java:218)
	at com.infy.business.service.ProductServiceImpl.getProductDetails(ProductServiceImpl.java:140)
	at com.infy.ui.UserInterface.getProductDetails(UserInterface.java:227)
	at com.infy.ui.UserInterface.main(UserInterface.java:43)
2018-08-01 05:05:38,857  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:197)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:135)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:124)
	at com.infy.ui.UserInterface.main(UserInterface.java:36)
2018-08-01 05:06:14,883  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:197)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:135)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:124)
	at com.infy.ui.UserInterface.main(UserInterface.java:36)
2018-08-01 05:07:06,113  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:197)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:135)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:124)
	at com.infy.ui.UserInterface.main(UserInterface.java:36)
2018-08-01 05:07:52,379  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:197)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:135)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:124)
	at com.infy.ui.UserInterface.main(UserInterface.java:36)
2018-08-01 05:08:08,775  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:197)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:135)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:124)
	at com.infy.ui.UserInterface.main(UserInterface.java:36)
2018-08-01 05:09:37,476  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:197)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:135)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:124)
	at com.infy.ui.UserInterface.main(UserInterface.java:36)
2018-08-05 11:15:46,312  ERROR [main] com.infy.business.service.ProductServiceImpl [getProductDetails] - Service.PRODUCT_NOT_FOUND
java.lang.Exception: Service.PRODUCT_NOT_FOUND
	at com.infy.business.service.ProductServiceImpl.getProductDetails(ProductServiceImpl.java:142)
	at com.infy.ui.UserInterface.getProductDetails(UserInterface.java:230)
	at com.infy.ui.UserInterface.main(UserInterface.java:45)
2018-08-16 06:05:23,969  ERROR [main] com.infy.dao.UserDAOImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:187)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:136)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:127)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-16 06:05:24,001  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:187)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:136)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:127)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-16 06:06:54,902  ERROR [main] com.infy.dao.UserDAOImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:187)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:136)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:127)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-16 06:06:54,934  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:187)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:136)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:127)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-16 06:08:05,572  ERROR [main] com.infy.dao.UserDAOImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:53)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:34)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:72)
	at com.infy.ui.UserInterface.main(UserInterface.java:34)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-08-16 06:08:05,588  ERROR [main] com.infy.business.service.UserServiceImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:53)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:34)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:72)
	at com.infy.ui.UserInterface.main(UserInterface.java:34)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-08-16 06:09:44,636  ERROR [main] com.infy.dao.UserDAOImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:53)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:34)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:72)
	at com.infy.ui.UserInterface.main(UserInterface.java:34)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-08-16 06:09:44,668  ERROR [main] com.infy.business.service.UserServiceImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:53)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:34)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:72)
	at com.infy.ui.UserInterface.main(UserInterface.java:34)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-08-16 06:09:53,071  ERROR [main] com.infy.dao.UserDAOImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:53)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:34)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:72)
	at com.infy.ui.UserInterface.main(UserInterface.java:34)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-08-16 06:09:53,087  ERROR [main] com.infy.business.service.UserServiceImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:53)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:34)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:72)
	at com.infy.ui.UserInterface.main(UserInterface.java:34)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-08-16 06:10:33,497  ERROR [main] com.infy.dao.UserDAOImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:187)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:136)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:128)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-16 06:10:33,513  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:187)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:136)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:128)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-16 06:11:52,839  ERROR [main] com.infy.dao.UserDAOImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:187)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:136)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:128)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-16 06:11:52,870  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:187)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:136)
	at com.infy.ui.UserInterface.findUser(UserInterface.java:128)
	at com.infy.ui.UserInterface.main(UserInterface.java:37)
2018-08-16 06:12:37,301  ERROR [main] com.infy.dao.UserDAOImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:187)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:136)
	at com.infy.business.service.UserServiceImpl.updateUser(UserServiceImpl.java:63)
	at com.infy.ui.UserInterface.updateUser(UserInterface.java:96)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
2018-08-16 06:12:37,316  ERROR [main] com.infy.business.service.UserServiceImpl [findUser] - 
java.lang.NullPointerException
	at com.infy.dao.UserDAOImpl.findUser(UserDAOImpl.java:187)
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:136)
	at com.infy.business.service.UserServiceImpl.updateUser(UserServiceImpl.java:63)
	at com.infy.ui.UserInterface.updateUser(UserInterface.java:96)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
2018-08-16 06:12:37,332  ERROR [main] com.infy.business.service.UserServiceImpl [updateUser] - Service.USER_NOT_FOUND
java.lang.Exception: Service.USER_NOT_FOUND
	at com.infy.business.service.UserServiceImpl.findUser(UserServiceImpl.java:143)
	at com.infy.business.service.UserServiceImpl.updateUser(UserServiceImpl.java:63)
	at com.infy.ui.UserInterface.updateUser(UserInterface.java:96)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
2018-09-16 05:39:32,979  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:266)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 05:40:24,183  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:266)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 05:41:23,810  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:266)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 05:51:03,830  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:266)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 05:55:57,290  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:266)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 05:56:51,797  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:266)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 05:57:30,015  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:266)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 05:58:19,352  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:266)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 05:58:46,800  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:267)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 05:59:00,371  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from INFYRETAIL_PRODUCT]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:267)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 06:05:50,413  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: * near line 1, column 8 [select * from com.infy.entity.ProductEntity]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: * near line 1, column 8 [select * from com.infy.entity.ProductEntity]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:267)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 06:07:16,042  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected char: ';' [select * from INFYRETAIL_PRODUCT;]
org.hibernate.QueryException: unexpected char: ';' [select * from INFYRETAIL_PRODUCT;]
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:250)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:267)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 06:07:45,201  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected char: ';' [select * from INFYRETAIL_PRODUCT;]
org.hibernate.QueryException: unexpected char: ';' [select * from INFYRETAIL_PRODUCT;]
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:250)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:267)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-09-16 06:55:36,199  DEBUG [main] com.infy.dao.ProductDAOImpl [getAllProductDetails] - unexpected token: from near line 1, column 8 [select from com.infy.entity.ProductEntity pe]
org.hibernate.hql.internal.ast.QuerySyntaxException: unexpected token: from near line 1, column 8 [select from com.infy.entity.ProductEntity pe]
	at org.hibernate.hql.internal.ast.QuerySyntaxException.convert(QuerySyntaxException.java:91)
	at org.hibernate.hql.internal.ast.ErrorCounter.throwQueryException(ErrorCounter.java:109)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.parse(QueryTranslatorImpl.java:304)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:203)
	at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:158)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:131)
	at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:93)
	at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:167)
	at org.hibernate.internal.AbstractSessionImpl.getHQLQueryPlan(AbstractSessionImpl.java:301)
	at org.hibernate.internal.AbstractSessionImpl.createQuery(AbstractSessionImpl.java:236)
	at org.hibernate.internal.SessionImpl.createQuery(SessionImpl.java:1836)
	at com.infy.dao.ProductDAOImpl.getAllProductDetails(ProductDAOImpl.java:269)
	at com.infy.business.service.ProductServiceImpl.getAllProductDetails(ProductServiceImpl.java:165)
	at com.infy.ui.UserInterface.getAllProductDetails(UserInterface.java:254)
	at com.infy.ui.UserInterface.main(UserInterface.java:47)
2018-12-05 03:58:06,262  ERROR [main] com.infy.resources.HibernateUtility [getSessionFactory] - Could not instantiate id generator [entity-name=com.infy.entity.ProductEntity]
org.hibernate.MappingException: Could not instantiate id generator [entity-name=com.infy.entity.ProductEntity]
	at org.hibernate.id.factory.internal.DefaultIdentifierGeneratorFactory.createIdentifierGenerator(DefaultIdentifierGeneratorFactory.java:123)
	at org.hibernate.mapping.SimpleValue.createIdentifierGenerator(SimpleValue.java:225)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:323)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1859)
	at com.infy.resources.HibernateUtility.getSessionFactory(HibernateUtility.java:30)
	at com.infy.resources.HibernateUtility.<clinit>(HibernateUtility.java:16)
	at com.infy.dao.ProductDAOImpl.addProduct(ProductDAOImpl.java:78)
	at com.infy.business.service.ProductServiceImpl.addProduct(ProductServiceImpl.java:66)
	at com.infy.ui.UserInterface.addProduct(UserInterface.java:185)
	at com.infy.ui.UserInterface.main(UserInterface.java:42)
Caused by: org.hibernate.MappingException: Could not interpret id generator strategy [INFYRETAIL_PRODUCT_SEQ]
	at org.hibernate.id.factory.internal.DefaultIdentifierGeneratorFactory.getIdentifierGeneratorClass(DefaultIdentifierGeneratorFactory.java:140)
	at org.hibernate.id.factory.internal.DefaultIdentifierGeneratorFactory.createIdentifierGenerator(DefaultIdentifierGeneratorFactory.java:114)
	... 9 more
2018-12-05 04:07:21,054  ERROR [main] com.infy.resources.HibernateUtility [getSessionFactory] - Could not instantiate id generator [entity-name=com.infy.entity.ProductEntity]
org.hibernate.MappingException: Could not instantiate id generator [entity-name=com.infy.entity.ProductEntity]
	at org.hibernate.id.factory.internal.DefaultIdentifierGeneratorFactory.createIdentifierGenerator(DefaultIdentifierGeneratorFactory.java:123)
	at org.hibernate.mapping.SimpleValue.createIdentifierGenerator(SimpleValue.java:225)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:323)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1859)
	at com.infy.resources.HibernateUtility.getSessionFactory(HibernateUtility.java:30)
	at com.infy.resources.HibernateUtility.<clinit>(HibernateUtility.java:16)
	at com.infy.dao.ProductDAOImpl.addProduct(ProductDAOImpl.java:78)
	at com.infy.business.service.ProductServiceImpl.addProduct(ProductServiceImpl.java:66)
	at com.infy.ui.UserInterface.addProduct(UserInterface.java:185)
	at com.infy.ui.UserInterface.main(UserInterface.java:42)
Caused by: org.hibernate.MappingException: Could not interpret id generator strategy [INFYRETAIL_PRODUCT_SEQ]
	at org.hibernate.id.factory.internal.DefaultIdentifierGeneratorFactory.getIdentifierGeneratorClass(DefaultIdentifierGeneratorFactory.java:140)
	at org.hibernate.id.factory.internal.DefaultIdentifierGeneratorFactory.createIdentifierGenerator(DefaultIdentifierGeneratorFactory.java:114)
	... 9 more
2018-12-11 11:02:41,905  ERROR [main] com.infy.dao.UserDAOImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:61)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:38)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:78)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-12-11 11:02:41,921  ERROR [main] com.infy.business.service.UserServiceImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:61)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:38)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:78)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-12-11 11:03:12,509  ERROR [main] com.infy.dao.UserDAOImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:61)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:38)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:78)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-12-11 11:03:12,540  ERROR [main] com.infy.business.service.UserServiceImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:61)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:38)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:78)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-12-11 11:04:51,516  ERROR [main] com.infy.dao.UserDAOImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:61)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:38)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:80)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-12-11 11:04:51,532  ERROR [main] com.infy.business.service.UserServiceImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:61)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:38)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:80)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-12-11 11:05:27,853  ERROR [main] com.infy.dao.UserDAOImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:61)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:38)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:80)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
2018-12-11 11:05:27,869  ERROR [main] com.infy.business.service.UserServiceImpl [addUser] - could not execute statement
org.hibernate.exception.ConstraintViolationException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:72)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:49)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:126)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:112)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:211)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:62)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3581)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:104)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:465)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:351)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1258)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:425)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at com.infy.dao.UserDAOImpl.addUser(UserDAOImpl.java:61)
	at com.infy.business.service.UserServiceImpl.addUser(UserServiceImpl.java:38)
	at com.infy.ui.UserInterface.addUser(UserInterface.java:80)
	at com.infy.ui.UserInterface.main(UserInterface.java:35)
Caused by: java.sql.SQLIntegrityConstraintViolationException: ORA-00001: unique constraint (SYSTEM.INFYRETAIL_USER_ID_PK) violated

	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:450)
	at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:399)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1059)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:522)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:257)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:587)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:225)
	at oracle.jdbc.driver.T4CPreparedStatement.doOall8(T4CPreparedStatement.java:53)
	at oracle.jdbc.driver.T4CPreparedStatement.executeForRows(T4CPreparedStatement.java:943)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1150)
	at oracle.jdbc.driver.OraclePreparedStatement.executeInternal(OraclePreparedStatement.java:4798)
	at oracle.jdbc.driver.OraclePreparedStatement.executeUpdate(OraclePreparedStatement.java:4875)
	at oracle.jdbc.driver.OraclePreparedStatementWrapper.executeUpdate(OraclePreparedStatementWrapper.java:1361)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:208)
	... 16 more
